src/styles.scss

```diff
@@ -5,3 +5,7 @@ body {
 
 // Import Bootstrap and its default variables
 @import '~bootstrap/scss/bootstrap.scss';
+
+body {
+  font-size: 87%;
```

Comment:
lol, `87`?

Reply:
This scales the default browser size (typically `16px`) to roughly `14px` which looked about right to me. Should I use a pixel value instead? The best practice used to be setting a percentage but iirc that was only because browsers wouldn't zoom pixel sized fonts. Now they do.

Reply:
I think we can keep it as is.